integration/gitopscreateprofile_test.go

```diff
@@ -0,0 +1,90 @@
+// +build integration
+
+package integration_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/spf13/afero"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+var _ = Describe("(Integration) generate profile", func() {
```

Comment:
Should we make it easy to run this in isolation from the rest of the suite?
Long-term, it'd be very nice to have parallel suites also :)

Reply:
Yes, How can I do that?

Reply:
I don't know, something that we will need to determine, we might want to play with focus options or tweak the main function of the integration suite... Perhaps best to get this in first?